ppr-ui/src/components/parties/party/RegisteringParty.vue

```diff
@@ -83,42 +87,46 @@
                       </v-list>
                     </v-menu>
                   </span>
-                </div>
-                </td>
-              </tr>
-              <tr v-if="showEditParty">
-              <td
-                colspan="5"
-                class="pa-0"
-              >
-                <v-expand-transition>
-                  <div class="edit-Party-container pa-0 col-12">
-                    <edit-party :setIsRegisteringParty="true" @resetEvent="resetData" />
-                  </div>
-                </v-expand-transition>
-              </td>
-            </tr>
-            </template>
-            <template  v-slot:no-data>
-              We were unable to retrieve Registering Party from your account. Please try
-              again later. If this issue persists, please contact us.
-              <br /><br />
-              <v-btn
-                id="retry-registering-party"
-                outlined
-                color="primary"
-                @click="getRegisteringParty()"
-              >
-                Retry <v-icon>mdi-refresh</v-icon>
-              </v-btn>
-              <error-contact class="search-contact-container pt-6" />
+                    </div>
```

Comment:
Can you double check the spacing here please

Reply:
Definitely, thank you!

Reply:
Cleaned up!